docker/pom.xml

```diff
@@ -34,10 +34,10 @@
   <packaging>bundle</packaging>
 
   <properties>
-    <test.docker.endpoint>https://localhost:4243</test.docker.endpoint>
+    <!--<test.docker.endpoint>${env.DOCKER_HOST}</test.docker.endpoint>-->
```

Comment:
It's now set by the property-regex maven plugin based on env.DOCKER_HOST. Can be removed so I did just that. :)